src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -768,6 +766,31 @@ async function copyCode(event: Event): Promise<void> {
 }
 
 const getIconColor = computed(() => props.theme === 'light' ? KUI_COLOR_TEXT_NEUTRAL_STRONG : KUI_COLOR_TEXT_INVERSE)
+
+type VirtualizerProps = InstanceType<typeof Virtualizer>['$props']
```

Comment:
nit: we typically define all types separately in the `types/` directory, or at least at the top of the script

May make sense here as it's next to the function; just resolve the comment if you'd prefer to keep it here